core/templates/pages/exploration-editor-page/services/exploration-data.service.ts

```diff
@@ -108,7 +108,7 @@ export class ExplorationDataService {
   async autosaveChangeListAsync(
       changeList: ExplorationChange[],
       successCallback: (response: DraftAutoSaveResponse) => void,
-      errorCallback = () => {}
+      errorCallback: () => void
```

Comment:
Arent you changing the behaviour here? Previous it was the default option.

Reply:
The function `autosaveChangeListAsync` is only used at a single place where we are passing error callback as an argument and not using a default value. Do we generally use default values like these in arguments ? This default also decreases the frontend coverage so I changed it to type. Here is it's only usage
![image](https://user-images.githubusercontent.com/60519303/125402017-8856ec00-e3d1-11eb-93a4-013443afe514.png)